exercises/isogram/isogram-example.apl

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env apl --script
+
+version ← '1.2.0' ⍝ isogram exercise, GNU-APL track, exercism
+
+alphabet ← 'abcdefghijklmnopqrstuvwxyz'
+ALPHABET ← 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+
+counts ← {+/ ⍺ ∘.= ⍵}
```

Comment:
As much as I like the idea of APL lambdas, I'd prefer that we not use them in Exercism sample code, since they're not part of the APL standard, and I would like to keep the sample code as close to the standard as possible to facilitate use of other APL interpreters than GNU APL.

In this case, it's easy enough to use `∇` syntax to define the function:

```apl
∇result←alphabet counts string
  +/ alphabet ∘.= string
∇
```